tensorflow/lite/micro/kernels/leaky_relu.cc

```diff
@@ -28,55 +28,59 @@ namespace micro {
 namespace activations {
 namespace {
 
+// Input/output tensor index.
+constexpr int kInputTensor = 0;
+constexpr int kOutputTensor = 0;
+
 // OLD-TODO(b/142762739): We should figure out a multi-threading plan for most
 // of the activation ops below.
 
-struct OpData {};
-
-struct LeakyReluOpData : public OpData {
-  int32_t output_multiplier_alpha = 0;
-  int32_t output_shift_alpha = 0;
-  int32_t output_multiplier_identity = 0;
-  int32_t output_shift_identity = 0;
+struct LeakyReluOpData {
+  // quantization parameters
+  int32_t output_multiplier_alpha;
+  int output_shift_alpha;
+  int32_t output_multiplier_identity;
+  int output_shift_identity;
```

Comment:
I think we should keep these as int32_t. We do have some non-32 bit targets as well.

Reply:
Fixed